lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -3,8 +3,8 @@ module Billing #:nodoc:
     class CredoraxGateway < Gateway
       class_attribute :test_url, :live_na_url, :live_eu_url
 
-      self.display_name = 'Credorax Gateway'
-      self.homepage_url = 'https://www.credorax.com/'
+      self.display_name = 'Finaro Gateway'
```

Comment:
I don't see any direct references to a "Finaro Gateway" in their site or docs, so for now let's retain this older name in AM - we actually use a different field in Core to display the name in  Docs anyway, so we can change it there.

```suggestion
      self.display_name = 'Credorax Gateway'
```